dist/index.d.mts

```diff
@@ -283,8 +283,8 @@ declare function new_wallet(seed?: any): {
  *      mnemonic:           Bip39 24 words mnemonic
  */
 declare function new_wallet_bip39(seed?: any, derivationIndex?: number): {
-    sk: string;
-    vk: string;
+    sk: any;
+    vk: any;
```

Comment:
why any ?

Reply:
Weird. I only formatted `src/lib/wallet.ts` with `prettier v3.2.5` and built it. I did not even touch any of your tsup configs